spec/elastic_apm/span_spec.rb

```diff
@@ -77,12 +81,12 @@ module ElasticAPM
         subject.stop
 
         expect(subject).to be_stopped
-        expect(subject.duration).to be 100_000
+        expect(subject.duration).to be 100
       end
     end
 
     describe '#done', :mock_time do
-      let(:duration) { 100 }
+      let(:duration_us) { 5_100 }
```

Comment:
what does `us` refer to?

Reply:
it's _micro_ seconds. The u is an ASCII version of a μ char.

Reply:
Ok, so you added that because you wanted to distinguish between the units here and the other references to `duration` (like the accessor references in the `transaction_spec`  ?

Reply:
Yes, I think I wanted it to purposefully look like just a value in the specs and not something coming from a trace.